docs/admin/configuration/radicale-integration.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 12
+id: radicale-integration
+title: Calendar and Contacts Integration with Radicale
+description: "How to enable Calendar and Contacts integration via Radicale in OpenCloud."
+---
+
+OpenCloud is able to act as an authenticating reverse proxy for the open-source
+CalDAV and CardDAV Server [Radicale](https://radicale.org).
```

Comment:
Maybe add something like 

With that, users get a calender- and contacts server out of the box with OpenCloud that they can use with a wide selection of desktop- and mobile clients without hassle.